applications/nrf5340_audio/src/modules/dfu_entry.c

```diff
@@ -6,12 +6,14 @@
 
 /* Override compiler definition to use size-bounded string copying and concatenation function */
 #define _BSD_SOURCE
-#include "string.h"
+#include "dfu_entry.h"
+
 #include <zephyr/kernel.h>
 #include <zephyr/bluetooth/bluetooth.h>
 #include <zephyr/bluetooth/conn.h>
 #include <zephyr/settings/settings.h>
 
+#include "string.h"
 #include "button_assignments.h"
 #include "ble_core.h"
```

Comment:
Maybe #include "ble_core.h" can be removed now?

Reply:
It should, that was the point of the task ;)

Reply:
done